Makefile

```diff
@@ -16,10 +16,6 @@ start-main:
 start-main-fast:
 	yarn run watch-main | yarn pino-pretty -c -l
 
-start-admin:
-	. ./.envrc && yarn tsnd --respawn --files -r tsconfig-paths/register -r src/services/tracing.ts \
-		src/servers/graphql-admin-server.ts | yarn pino-pretty -c -l
```

Comment:
`src/servers/graphql-admin-server.ts` still is not deleted

Reply:
`start-main` can then be renamed to `start`